src/TextArea/TextArea.tsx

```diff
@@ -179,6 +180,7 @@ export class TextArea extends React.Component<Props, State> {
     const inputNote = this.renderNote(errorMessage);
 
     const textAreaProps = {
+      ["aria-invalid"]: !valid,
```

Comment:
Why do we need to set `aria-invalid`?

Reply:
This article is referenced in the JIRA story: https://hiddedevries.nl/en/blog/2017-04-04-how-to-make-inline-error-messages-accessible